features/steps/nested_folders_for_steps.py

```diff
@@ -0,0 +1,23 @@
+from behave import then, when
+
+
+@when('a step is implemented in the root folder')
+def step_impl(context):
+    context.nested_root_visited = True
+
```

Comment:
The linting and test don't run anymore. There's something wrong. Please make sure checks and tests run.

```suggestion


```

Here, there should be two lines between the first, the second and the third function, instead of only one.